ironfish/src/consensus/verifier.ts

```diff
@@ -183,6 +183,9 @@ export class Verifier<
     } catch {
       return Promise.reject('Could not deserialize transaction')
     }
+    if ((await transaction.transactionFee()) < 0) {
+      return Promise.reject('Transaction has negative fees')
+    }
```

Comment:
note: this needs to move to the mem accept() function later

Reply:
I'd like to leave it here as well in the future, but yep, we still need to prevent readding the transaction to the mempool and broadcasting it